lucene/misc/src/test/org/apache/lucene/misc/index/TestIndexSplitter.java

```diff
@@ -60,9 +67,9 @@ public void test() throws Exception {
       iw.addDocument(doc);
     }
     iw.commit();
-    DirectoryReader iwReader = iw.getReader();
-    assertEquals(3, iwReader.leaves().size());
-    iwReader.close();
+    //DirectoryReader iwReader = iw.getReader();
+    //assertEquals(3, iwReader.leaves().size());
+    //iwReader.close();
```

Comment:
IndexWriter.getReader() is package-private so it cannot be accessed from here. It would be better that we omit this assertion (it seems these lines are not the main part of the test anyway), instead of opening IW.getReader() up to public?

Reply:
Or maybe switch to DirectoryReader.open(iw...) and keep the assertion? This is public.

Reply:
Thanks! DirectoryReader.open(iw) and IW.getReader() are actually equivalent. https://github.com/apache/lucene-solr/pull/2064/commits/4858a5d8347c72364c4a6572345a3fc52f43cc77